docs/glossary.rst

```diff
@@ -0,0 +1,54 @@
+.. -*- coding: utf-8-with-signature -*-
+
+=======================
+Glossary of Tahoe terms
+=======================
+
+
+.. glossary::
+
+    cap
+    capability
+        dircap, filecap, write cap, read cap, verify cap; all these refer to a capability or privilege associated with a key.
+        The word "Capability" comes from "Object Capability" or OCap theory, in the manner of `the E language <http://erights.org/elib/capability/ode/ode-capabilities.html>`_.
+        In the context of Tahoe-LAFS these capability objects are somewhat short strings of data giving the holder necessary and sufficient secret information to carry out the operation and turn the capability into a less-capable one.
+        For example, a "verify cap" gives only enough information to confirm the ciphertext is available (but not to decode or read it); a "read cap" can decode and read the data (but not modify it) and can be turned into a "verify cap".
+
+    client
+        Be aware that the term "client" is ambiguously used as a "client (to gateway) program" and as a gateway node. It generally refers to the network process for a data or control plane.
+
+    foolscap
+        Foolscap is an RPC/RMI (Remote Procedure Call / Remote Method Invocation) protocol for use with Twisted, derived/inspired by Twisted's built-in "Perspective Broker" package. https://github.com/warner/foolscap
+
+    fURL
+        A Foolscap URL. A Foolscap connection setup takes as an input.
+
+    helper
+        The “Helper” is a service that can mitigate the expansion penalty by arranging for the client node to send data to a central Helper node instead of sending it directly to the storage servers. :doc:`Helper Overview </helper>`
+
+    introducer
+        A collection of Tahoe servers is called a Grid and usually has 1 Introducer (but sometimes more, and it’s possible to run with zero). The Introducer announces which storage servers constitute the Grid and how to contact them. There is a secret “fURL” you need to know to talkto the Introducer.
```

Comment:
Should probably add "Grid" to the glossary too.

This is a bit of a tough one as "how is a collection defined" becomes a little tricky, but ...

Reply:
Thank you, yes grid is in a later version and I'm sure that there's room for improvement.